src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -483,6 +519,50 @@
     type: group
     short-summary: Manage deployment operations.
 """
+helps['group lock'] = """
+    type: group
+    short-summary: Manage Azure resource group locks.
+"""
+helps['group lock create'] = """
+    type: command
+    short-summary: Create a resource group lock.
+    examples:
+        - name: Create a read-only resource group level lock.
+          text: >
+            az account lock create --lock-type ReadOnly -n lockName -g MyResourceGroup
```

Comment:
This is a copy and paste error. Please update a few places below

Reply:
Thanks for catching that